game-engine/default-config.json

```diff
@@ -40,11 +58,14 @@
     "dig": 7,
     "move": 5,
     "powerup": 20,
+    "freeze": 17,
     "doNothing": 0,
     "invalidCommand": -4
   },
   "wormSelectTokens": {
     "count": 5
   },
-  "csvSeparator": ","
+  "csvSeparator": ",",
+  "battleRoyaleStart": 0.25,
+  "battleRoyaleEnd": 0.875
```

Comment:
Shouldn't this rather be specified in rounds?

Reply:
That would make these dependent on the maxRounds number (if these are bigger than, they won't do anything then...i think)